src/components/ArticleFooter.js

```diff
@@ -0,0 +1,49 @@
+import { h } from 'preact'
+import { useState, useEffect } from 'preact/hooks'
+import { useI18n } from 'hooks'
+import { getArticleFooter } from 'api'
+
+// The reason why adding placeholder items is
+// the use article links navigation able to select the first item
+const PLACEHOLDER_ITEMS = [{ title: 'loading...' }, { title: 'loading...' }, { title: 'loading...' }]
+
+export const ArticleFooter = ({ lang, title }) => {
+  const [footer, setFooter] = useState(PLACEHOLDER_ITEMS)
+  const i18n = useI18n()
+
+  useEffect(() => {
+    getArticleFooter(lang, title)
+      .then(items => setFooter(items))
+  }, [])
+
+  return (
+    <div class='article-footer'>
+      <div class='content'>
+        <h2>{i18n.i18n('suggested-articles')}</h2>
+        <div class='list'>
+          { footer.map(item => {
+            return <a class='item' title={item.title}>
+              <div class='info'>
+                <div class='article-title'>{item.title}</div>
+                <div class='description'>{item.description}</div>
+              </div>
+              { item.thumbnail && <div class='img'><img src={item.thumbnail.source} /></div> }
+            </a>
+          })}
+        </div>
+        <h2>
+          <img src='/images/wikipedia-wordmark-en.png' height='18' width='116' />
+        </h2>
+        <p class='license' dangerouslySetInnerHTML={
+          { __html: i18n.i18n('content-license', '<a class=\'external\' rel=\'mw:ExtLink\' href=\'https://creativecommons.org/licenses/by-sa/3.0/\'>CC BY-SA 3.0</a>') }
```

Comment:
Building this link externally makes it untranslatable. In this case the link should be included in the message. 

See https://github.com/wikimedia/mediawiki-extensions-WikimediaMessages/blob/master/i18n/wikimedia/en.json#L217